src/components/registration/RegistrationTable.jsx

```diff
@@ -1,27 +1,89 @@
-import React from "react";
+import React, { useContext } from "react";
 import MUIDataTable from "mui-datatables";
 import PropTypes from "prop-types";
+import { FieldsContext } from "../../context/fields";
+import QuestionTypes from "../../constants";
 
-const columns = [
+const options = {
+  responsive: "standard",
+  selectableRows: "none",
+};
+
+const defaultColumns = [
```

Comment:
Any way we can store these magic strings in a const or another file? Just in case we need to update names in the future 😎